src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -3029,6 +3029,11 @@ public static void createMissingStaticSitePins(BELPin belPin, SiteInst si, Cell
                     net = si.getDesign().getGndNet();
                 }
             }
+            String belName = belPin.getBELName();
+            if (DesignTools.isBELALut(belName) && si.getCell(belName.replace('6', '5')) == null) {
```

Comment:
```suggestion
            String belName = belPin.getBELName();
            if (bel.isLUT() && si.getCell(belName.replace('6', '5')) == null) {
```

Reply:
I don't think the `bel` variable exists.